lms/djangoapps/courseware/models.py

```diff
@@ -233,6 +234,47 @@ def get_history(student_modules):
 
         return history_entries
 
+    @staticmethod
+    def save_history_entry(sender, instance, history_model_cls, request_cache_key, **kwargs):
```

Comment:
This signature still looks like it's a signal receiver, but it's not, right?

1. Can we get rid of `sender` since it doesn't appear to be used?
2. Please substitute `instance` with `student_module` so it's clearer what kind of thing it is.
3. Is it possible to get rid of `kwargs`?